internal/buildscripts/packaging/installer/install.sh

```diff
@@ -379,6 +405,7 @@ install() {
         install_td_agent_apt_repo "$td_agent_version"
         apt-get -y update
         install_apt_package "td-agent" "$td_agent_version"
+        apt-get -y install build-essential libcap-ng0 libcap-ng-dev pkg-config
```

Comment:
is `libcap-ng0` required? is the `0` expected?

Reply:
According to https://docs.fluentd.org/deployment/linux-capability, both the `libcap-ng` and the dev packages are required.  Installing the dev package usually installs the main package, but I include it just in case.  The `0` in the debian package name is correct.

Reply:
Yes, both are required, but dev has a dependency on the main package. Seems fine to be explicit regardless.